lib/ffi/src/lib.rs

```diff
@@ -401,3 +475,120 @@ pub extern "C" fn rv_render(ctx: &mut Context, viewport: Rect) {
 
     ctx.queue.submit(Some(encoder.finish()));
 }
+
+fn vec_i2f(v: [i32; 3]) -> [f32; 3] {
+    [v[0] as f32, v[1] as f32, v[2] as f32]
+}
+
+#[no_mangle]
+pub extern "C" fn rn_model_create(
```

Comment:
Should it be `rv_` names instead `rn_`?

Reply:
funny how errors proliferate and multiply themselves!